src/reducers/productosReducer.js

```diff
@@ -0,0 +1,14 @@
+//Cada reducer tiene su propio state
+
+const initialState = {
+  productos: [],
+  error: null,
+  loading: false,
+};
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    default:
+      return state;
+  }
+}
```

Comment:
A suggestion is install redux-actions library or immer to handler the reducer state and mutation